### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- fix linter warnings

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -122,23 +111,23 @@ ynh_system_user_create --username=$app
 # CREATE DIRECTORIES
 #=================================================
 
-mkdir -p /home/yunohost.$app/{Podcasts,Playlists}
+mkdir -p /home/yunohost.app/$app/{Podcasts,Playlists}
```
Comment (Gofannon) [inline scripts/upgrade:114]: With this new folder, what happen to people who have their files in the old one?

You should move the content of the old one to the new one?
Maybe also removing the old one but in the case, what happen what the airsonic behavior?

File: scripts/upgrade [hunk 2]
```diff
@@ -200,14 +189,7 @@ ynh_script_progression --message="Upgrading systemd configuration..." --weight=1
...
 
-ynh_backup_if_checksum_is_different --file="$service_config"
-
-# Copy config. template to right location
-cp ../conf/systemd-sysconfig $service_config
-
-ynh_replace_string --match_string="__FINALPATH__" --replace_string=$final_path --target_file="$service_config"
-ynh_replace_string --match_string="__PORT__" --replace_string=$port --target_file="$service_config"
-ynh_replace_string --match_string="__PATH__" --replace_string=$path_url --target_file="$service_config"
+ynh_add_config --template="../conf/systemd-sysconfig" --destination="/etc/default/$app"
```
Comment (Gofannon) [inline scripts/upgrade:192]: I don'nt much about the helper, I will trust you on this. Seems much cleaner !

File: scripts/upgrade [hunk 3]
```diff
@@ -19,10 +19,7 @@ app=$YNH_APP_INSTANCE_NAME
 domain=$(ynh_app_setting_get --app=$app --key=domain)
 path_url=$(ynh_app_setting_get --app=$app --key=path)
 admin=$(ynh_app_setting_get --app=$app --key=admin)
-is_public=$(ynh_app_setting_get --app=$app --key=is_public)
-final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 port=$(ynh_app_setting_get --app=$app --key=port)
-service_config=$(ynh_app_setting_get --app=$app --key=service_config)
```
Comment (Gofannon) [inline scripts/upgrade:25]: You should remove this variable if it's not used anymore

File: check_process [hunk 4]
```diff
@@ -4,6 +4,7 @@
 		path="/path"	(PATH)
 		admin="john"	(USER)
 		is_public=1	(PUBLIC|public=1|private=0)
+		port="666"	(PORT)
```
Comment (Gofannon) [inline check_process:7]: I don't think it's used in the bash scripts anywhere. Maybe a quick copy/paste?

File: check_process [hunk 5]
```diff
@@ -20,3 +21,7 @@
 ;;; Options
 Email=
 Notification=none
+;;; Upgrade options
```
Comment (Gofannon) [inline check_process:24]: copy/paste too?

File: scripts/install [hunk 6]
```diff
@@ -6,6 +6,10 @@
 # IMPORT GENERIC HELPERS
 #=================================================
 
+if [ ${PACKAGE_CHECK_EXEC:-0} -eq 1 ]; then
```
Comment (Gofannon) [inline scripts/install:9]: Needed for the CI I suppose?

**Comment by ericgaspar** [comment]
!testme

File: check_process [hunk 7]
```diff
@@ -15,8 +15,11 @@
...
 		backup_restore=1
 		multi_instance=1
-		port_already_use=0
 		change_url=1
 ;;; Options
 Email=
 Notification=none
+;;; Upgrade options
+	; commit=72d530415c016b23b6ba62085272957b04d2cca6
+		name=Name and date of the commit.
```
Comment (alexAubin) [inline check_process:24]: Ideally would be nice to give a proper name to the commit :stuck_out_tongue_winking_eye:

File: scripts/upgrade (hunk 1 above)
Comment (alexAubin) [reply scripts/upgrade:114]: +1, I think we can add : 

```bash
# Rename legacy folder to proper location
if [[ ! -e /home/yunohost.app/$app ]] && [[ -e /home/yunohost.$app ]]
then
    mkdir -p /home/yunohost.app/
    mv /home/yunohost.$app /home/yunohost.app/$app
fi
```

**Comment by alexAubin** [comment]
!testme

**Comment by alexAubin** [comment]
(:heavy_check_mark: approval to merge if tests are passing)

**Comment by alexAubin** [comment]
Yolomergin'

ezgif-5-a2085414a9

**Pull request merged**